README.md

```diff
@@ -29,19 +29,22 @@ As the current Microsoft Teams Graph APIs are only accessible by a tenant admin,
 
 You'll need to register an app through the following process:
 
-1. Sign into the [App Registration Portal](https://apps.dev.microsoft.com) using your personal, work or school account.
-2. Choose 'Add an app'.
-3. Enter a name for the app, and choose 'Create application'.
-4. The registration page displays, listing the properties of your app.
-   * Copy the Application Id. This is the unique identifier for your app.
-5. Under 'Application Secrets', choose 'Generate New Password'.
-   * Copy the password from the 'New password generated' dialog.
-6. Under 'Platforms', choose 'Add platform'.
-7. Choose 'Web'.
-8. Make sure 'Allow Implicit Flow' check box is selected, and enter 'Redirect URI' e.g., http://localhost:55065/.  See appropriate sample for more information on the specific port.
-   * The 'Allow Implicit Flow' option enables the hybrid flow. During authentication, this enables the app to receive both sign-in info (the id_token) and artifacts (in this case, an authorization code) that the app can use to obtain an access token.
-9. Under 'Microsoft Graph Permissions', Add 'Group.ReadWrite.All' (Read and write all groups) and 'User.ReadWrite.All' (Read and write all users' full profile) as Delegated and Application Permissions.
-10. Choose Save.
+1. Sign in to the [Azure portal](https://portal.azure.com/) using either a work or school account or a personal Microsoft account.

```

Comment:
@rwike77 Can you please modify to tell the user to go to Azure portal app registrations via this direct link? https://go.microsoft.com/fwlink/?linkid=2083908

This way instead of directing the user to navigate UI, which will change soon with portal GA, we can link them directly to the new app registrations section. So Step 3 can be deleted.

Reply:
@Jackson-Woods  OK, I updated the link in step 1.  Step 3 is a lot simpler now.  How's that look?

Reply:
@rwike77 Perfect, thanks. Merging.